launch_testing/launch_testing/launch_test.py

```diff
@@ -98,7 +98,7 @@ def main():
         parser.error("Test file '{}' does not exist".format(args.test_file))
 
     args.test_file = os.path.abspath(args.test_file)
-    test_module = _load_python_file_as_module(args.test_file)
+    test_module = _load_python_file_as_module(args.test_name, args.test_file)
```

Comment:
I guess this works too - usually the only time we care about the test name is when we run in CI and there we have launch_testing_cmake to give us a good test name.

For reference, https://github.com/ApexAI/apex_rostest/blob/c8cd0ac6dc5383334fdbfd67e7bfeee62bf37c1f/apex_launchtest/apex_launchtest/apex_launchtest_main.py#L34
I think this is a result of the parametrization stuff getting grabbed while it was still WIP